src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,47 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
-        List<Product> products = getAllProductsFromShoppingCart(userId);
+        logger.info("completeOrder method was called for userId: {}", userId);
+        List<Product> products = null;
+        try {
+            products = getAllProductsFromShoppingCart(userId);
+            logger.debug("Products retrieved from shopping cart for userId: {}: {}",
+                    userId, products);
+        } catch (Exception e) {
+            logger.error("Failed to retrieve products from shopping cart for userId: {}",
+                    userId, e);
+        }
```

Comment:
It's good that you're logging the exception, but it's also important to handle it appropriately. Simply logging and continuing as if nothing happened may not be the best approach. Consider adding some form of error handling, such as rethrowing the exception, returning an error status, or providing a fallback mechanism.